### DISCUSSION TITLE
Download the Script link broken on "build_term_tables" page

### DISCUSSION THREAD

**Discussion by ericwenger-pm** [nih-cfde/published-documentation#210] [Bug] [answered]
The "Download the Script" link appears to be broken on the "build_term_tables" page, https://github.com/nih-cfde/published-documentation/wiki/build_term_tables 

Broken link:
https://github.com/nih-cfde/published-documentation/wiki/build_term_tables#1-download-the-script

**Comment by ACharbonneau** [comment, marked as answer]
Thanks! That's super weird, the site is suddenly trying to render the script as a html page instead of downloading. It looks like the table downloads are still working tho. I'm going to have to work out how to fix the linking. In the meantime, I'm attaching the script here. 
I had to compress it to attach it, so you'll need to unzip it to run.

[build_term_tables.py.zip](https://github.com/nih-cfde/published-documentation/files/6213409/build_term_tables.py.zip)

**Reply by ACharbonneau** [reply]
This should be fixed.
